api/db/migrate/20210902162826_create_spree_webhooks_tables.rb

```diff
@@ -0,0 +1,30 @@
+class CreateSpreeWebhooksTables < ActiveRecord::Migration[6.1]
+  def change
+    create_table :spree_webhooks_events do |t|
+      t.text :name, null: false
+      t.boolean :enabled, default: true
+
+      t.timestamps
+    end
+
+    create_table :spree_webhooks_endpoints do |t|
+      t.text :url, null: false
+      t.boolean :enabled, default: true
```

Comment:
```suggestion
      t.boolean :enabled, default: false
```
by default these should be disabled, it's a safer approach